Cargo.toml

```diff
@@ -65,14 +66,14 @@ futures = "0.3.28"
 bytes = "1.4.0"
 dashmap = "5.4.0"
 redis = { version = "0.23.0", features = ["tokio-comp"], optional = true }
-surrealdb = { version = "1.0.0-beta.9+20230402", optional = true }
+surrealdb = { optional = true }
 
 [dev-dependencies]
 axum = { version = "0.6.12", features = ["macros"]}
 hyper = "0.14.25"
 tower = "0.4.13"
 log = { version = "0.4.17", default-features = false }
 
-[package.metadata.docs.rs]
-features = ["sqlite-rustls","mysql-rustls","redis-db", "surrealdb-rocksdb", "surrealdb-mem"]
-rustdoc-args = ["--document-private-items"]
+# [package.metadata.docs.rs]
```

Comment:
because of this warning
`warning: /Users/Hos/wk/AxumSessions/Cargo.toml: version requirement `1.0.0-beta.9+20230402` for dependency `surrealdb` includes semver metadata which will be ignored, removing the metadata is recommended to avoid confusion`

Reply:
leave them enabled. the warning is fine to see. otherwise the docs.rs documents wont contain redis and the rest when published to crates.io.